virtualizarr/manifests/manifest.py

```diff
@@ -141,6 +141,15 @@ def __init__(self, entries: dict) -> None:
         self._offsets = offsets
         self._lengths = lengths
 
+    @classmethod
+    def empty(cls) -> "ChunkManifest":
+        """Create an empty chunk manifest."""
+        return cls.from_arrays(
+            paths=np.array((), dtype=np.dtypes.StringDType),
+            offsets=np.array((), dtype="uint64"),
+            lengths=np.array((), dtype="uint64"),
+        )
```

Comment:
Why do we need this? Why can't we just use `ChunkManifest.__init__` with `entries={}`?

Reply:
that fails because of the list unpacking in `validate_chunk_keys`. It would probably be possible to fix/work around that, but the separate class method makes the intent very clear.

Reply:
I think I would prefer not to have a new constructor method just for a special case of the general `__init__` method. I would rather have an if...else... inside `validate_chunk_keys`. I feel like the intent of `ChunkManifest(entries={}` is also pretty clear.

Reply:
done